src/Magnum/Audio/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ set(MagnumAudio_HEADERS
 
     visibility.h)
 
+if(WITH_SCENEGRAPH)
```

Comment:
This is great. Exactly what I wanted to note but then I saw the conditional compilation is already done :)